homeassistant/components/backup/manager.py

```diff
@@ -60,45 +60,10 @@ class BaseBackupManager(abc.ABC):
     def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the backup manager."""
         self.hass = hass
-        self.backups: dict[str, Backup] = {}
         self.backing_up = False
-
-    async def async_post_backup_actions(self, **kwargs: Any) -> None:
-        """Post backup actions."""
-
-    async def async_pre_backup_actions(self, **kwargs: Any) -> None:
-        """Pre backup actions."""
```

Comment:
These 2 didn't move, is that intentional?

Reply:
<https://github.com/home-assistant/core/blob/bakup-load/homeassistant/components/backup/manager.py#L86-L116> they did?
Or rather, they were removed, and replaced.

Reply:
I meant that they are not moved in this PR. So all good 👍

Reply:
They were, github just shows the diff strangely for some reason 👍